_ml-commons-plugin/pretrained-models.md

```diff
@@ -39,16 +39,16 @@ The model-serving framework supports the following models, categorized by type.
 
 Sentence transformer models map sentences and paragraphs across a dimensional dense vector space. The number of vectors depends on the model. Use these models for use cases such as clustering and semantic search. 
 
-The following table provides a list of sentence transformer models and artifact links to download them:
+The following table provides a list of sentence transformer models and artifact links to download them. As of OpenSearch 2.6, all artifacts are set to version 1.0.1.
 
-| **Model name** | **Vector dimensions** | **Torchscript artifact** | **ONNX artifact** |
+| **Model name** | **Vector dimensions** | **Auto-truncation** | **Torchscript artifact** | **ONNX artifact** |
```

Comment:
Should we also add max token size for truncation? @dhrubo-os

Reply:
Discussed offline. We will not add max token size for now.